src/transformState.js

```diff
@@ -1,11 +1,36 @@
-'use strict';
+"use strict";
```

Comment:
Be consistent with your usage of quotes. In the previous commit, you had single quotes, now you have double quotes. Also, we have a 'single-quote' eslint rule in this project.